homeassistant/components/easyenergy/__init__.py

```diff
@@ -13,9 +13,11 @@
 from .coordinator import EasyEnergyDataUpdateCoordinator
 from .services import async_setup_services
 
-PLATFORMS = [Platform.SENSOR]
+PLATFORMS: list[Platform] = [Platform.SENSOR]
 CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
 
+type EasyEnergyConfigEntry = ConfigEntry[EasyEnergyDataUpdateCoordinator]
+
```

Comment:
I suggest that you move this to `coordinator.py`, so that you can use the new type on line 36.